### DISCUSSION TITLE
Colorized filetypes

### DISCUSSION THREAD

**Discussion by og900aero** [sxyazi/yazi#498] [1. Q&A]
Is it possible to color not only based on mime types, but also based on file types?
Example:
- exec (have +x permission)
- link
- orphan
- block
- sticky
- fifo
- sock

**Comment by sxyazi** [comment]
Duplicate of https://github.com/sxyazi/yazi/issues/262, closing this in favor of that issue.

**Comment by sxyazi** [comment]
Done by https://github.com/sxyazi/yazi/pull/510
